### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by pavel-mironchik**

### CODE REVIEW

**Comment by pavel-mironchik** [comment]
[Dashboard Page](https://trello.com/c/kPxi9p6t/60-dashboard-page)

**Comment by pavel-mironchik** [comment]
[Audit Log](https://trello.com/c/CXs8nfc8/39-audit-log)

File: admin/src/views/Dashboard.vue [hunk 1]
```diff
@@ -267,133 +269,112 @@
...
+              <template v-slot:item.user="{ value }">
+                {{ value.name }}
+              </template>
+              <template v-slot:item.auditable_type="{ item }">
+                <router-link
+                  class="text-no-wrap"
+                  v-if="item.id"
+                  :to="{ name: getEntityClass(item.auditable_type), params: { id: item.id } }"
+                >
+                  {{ getEntityClass(item.auditable_type) }} {{ item.id }}
```
Comment (alquist42) [inline admin/src/views/Dashboard.vue:345]: i think `item.id` here is the audit record it, maybe `auditable_id`?

Comment (pavel-mironchik) [reply admin/src/views/Dashboard.vue:345]: Yeah, you're right. I'm pushing the fix.

**Pull request merged**